prep-javascript-variables/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    const fullName = 'Nathan Lam';
+    console.log('fullName:', fullName);
+
+    const numberOfPets = '4';
+    console.log('numberOfPets:', numberOfPets);
+
+    const likesVegetables = 'true';
+    console.log('likesVegetables:', likesVegetables);
+
+    const nothing = 'null';
+    console.log('nothing:', nothing);
```

Comment:
For numbers, booleans, and `null`, you should leave out the quotes, because otherwise JavaScript sees a string.